example/basic.md

```diff
@@ -49,6 +62,62 @@ document.addEventListener('DOMContentLoaded', () => {
 });
 ```
 
+</div>
+<div class="tab-pane fade" id="legacy" role="tabpanel" aria-labelledby="legacy-tab">
+
+```HTML
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="https://storage.googleapis.com/crisp-store-locator/${package_version}/store-locator.css" />
+    <title>Store Locator Example</title>
+
+    <script type="text/javascript">
+      document.addEventListener('DOMContentLoaded', function () {
+        window["storeLocator"].createStoreLocatorMap({
```

Comment:
Just curious, is there a reason why the bracket notation is necessary here (I know this would be in TS)? I think it should be fine to use `window.storeLocator` directly in JS. Either way is fine, I only thought the latter is more customary.

Reply:
Either seems to work. I think this was mostly just leftover from things I was trying. I agree `window.storeLocator` seems more common!